src/components/ChataTable/TableWrapper.js

```diff
@@ -85,6 +85,7 @@ export default class TableWrapper extends React.Component {
   instantiateTabulator = () => {
     // Instantiate Tabulator when element is mounted
     this.tabulator = new Tabulator(this.tableRef, {
+      debugInvalidOptions: false,
```

Comment:
Is this the right solution to the problem of those console logs warnings showing up? Or should could we handle the invalid options better?